.github/workflows/migration-test.yml

```
@@ -121,4 +121,29 @@ jobs:
+            --schema-only \
+            --no-comments \
+            --no-owner \
+            --quote-all-identifiers \
+            -T public.goose_db_version \
+            -T public.goose_db_version_id_seq | sed \
+            -e '/^--.*/d' \
+            -e '/^SET /d' \
+            -e '/^SELECT pg_catalog./d' \
+            -e 's/"public"\.//' | awk '/./ { e=0 } /^$$/ { e += 1 } e <= 1' \
+            > ./current_schema.sql
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)